app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -6,14 +6,14 @@ import androidx.lifecycle.ViewModel
 import com.google.firebase.auth.FirebaseUser
 import edu.ucandroid.weathernotice.dto.Weather
 import edu.ucandroid.weathernotice.service.WeatherService
-import com.google.firebase.firestore.FirebaseFirestore
 import edu.ucandroid.weathernotice.dto.LocationInfo
 import edu.ucandroid.weathernotice.dto.Reminder
+import edu.ucandroid.weathernotice.service.FirestoreService
 import edu.ucandroid.weathernotice.service.LocationService
 
 class MainViewModel : ViewModel() {
 
-    private lateinit var firestore : FirebaseFirestore
+    private var firestore : FirestoreService = FirestoreService()
```

Comment:
Made "firestore" a "FirestoreService" object instead of a "FirebaseFirestore" object.

Reply:
Thank you will attempt to try this.